addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/CookieHttpOnlyScanRule.java

```diff
@@ -134,4 +135,30 @@ private Model getModel() {
     protected void setModel(Model model) {
         this.model = model;
     }
+
+    private int getCweId() {
+        // CWE Id 16 - Configuration
+        return 16;
+    }
+
+    private int getWascId() {
+        // WASC Id 13 - Info leakage)
+        return 13;
+    }
+
+    public List<Alert> getExampleAlerts() {
+        List<Alert> alerts = new ArrayList<Alert>();
+        Alert alert =
+                newAlert()
```

Comment:
This should call existing method, avoids maintaining the same info in two places (which is error prone). There's another PR with similar changes: https://github.com/zaproxy/zap-extensions/pull/2517/files#diff-2587ce4776bf99dbb14cf9284389ba13
The methods `getCweId()` and `getWascId()` would no longer be needed.